bcc/perf.go

```diff
@@ -180,3 +180,16 @@ func (pm *PerfMap) poll(timeout int) {
 		}
 	}
 }
+
+func bpfOpenPerfBuffer(cpu uint, callbackDataIndex uint64) (unsafe.Pointer, error) {
+	cpuC := C.int(cpu)
+	reader, err := C.bpf_open_perf_buffer(
+		(C.perf_reader_raw_cb)(unsafe.Pointer(C.callback_to_go)),
+		(C.perf_reader_lost_cb)(unsafe.Pointer(nil)),
```

Comment:
Isn't just `nil` enough?

Reply:
@schu Done. I changed it simply to `nil` without perf_reader_lost_cb.